eventfiles/models.py

```diff
@@ -17,8 +17,17 @@ def validate_unique_file(up_file, up_participant_id):
         if EventFile.objects.filter(file=up_file, participant_id=up_participant_id):
             raise ValidationError('that file already exist in meeting')
 
+    @staticmethod
+    def get_files_by_event(event_id):
```

Comment:
Can you move this function -> get_files_by_event
into a manager of event file.

An example of this can be found here:
https://github.com/redhat-beyond/lets-meet/blob/04f6afac8f75c3ca2d24a701b9f2aee9feed885b/events/class_models/event_models.py#L7-L10

Reply:
Done